tools/sdk-ai-bots/Scripts/Common.ps1

```diff
@@ -31,6 +31,39 @@ function Clone-Repository {
     return $true
 }
 
+function Test-AzCopyInstalled {
+    try {
+        $azcopyCommand = Get-Command azcopy -ErrorAction Stop
+        if ($azcopyCommand) {
+            return $true
+        }
+    }
+    catch {
+        Write-Error "AzCopy is not installed."
+    }
+    return $false
+}
+
+function Download-AzCopy {
+    param (
+        [Parameter(Position = 0)]
+        [ValidateNotNullOrEmpty()]
+        [string] $DestinationPath
+    )
+
+    try {
+        $AzCopyUrl = "https://azcopyvnext.azureedge.net/release20220315/azcopy_windows_amd64_10.14.1.zip"
```

Comment:
This is also a fixed value for the location, will it change in the future and can we use a parameter with default value?

Reply:
This URL will only be used locally and won't be used in pipeline scenario.